prosody/rootfs/etc/cont-init.d/10-config

```diff
@@ -1,5 +1,13 @@
 #!/usr/bin/with-contenv bash
 
+if [[ ! -f /config/saslauthd.conf ]]; then
+  cp /defaults/saslauthd.conf /config/
+  tpl /config/saslauthd.conf > /etc/saslauthd.conf
```

Comment:
If I restart the container the `/config/saslauthd.conf` does exists while `/etc/saslauthd.conf` not.

Looks like u have to split the condition here.

Reply:
Hi @jkroepke.
Fixed, try to use d336f39

Reply:
rebased in 14996e0